docker/cajtck.sh

```diff
@@ -48,24 +54,23 @@ cd $TS_HOME/bin
 sed -i "s#^build.level=.*#build.level=2#g" ts.jte
 sed -i "s#^endorsed.dirs=.*#endorsed.dirs=$TCK_HOME/glassfish5/glassfish/modules/endorsed#g" ts.jte
 sed -i "s#^local.classes=.*#local.classes=$TCK_HOME/glassfish5/glassfish/modules/endorsed/jakarta.annotation-api.jar#g" ts.jte
-sed -i "s#^report.dir=.*#report.dir=$TCK_HOME/cajtckreport/cajtck#g" ts.jte
-sed -i "s#^work.dir=.*#work.dir=$TCK_HOME/cajtckwork/cajtck#g" ts.jte
+sed -i "s#^report.dir=.*#report.dir=$TCK_HOME/${TCK_NAME}report/${TCK_NAME}#g" ts.jte
+sed -i "s#^work.dir=.*#work.dir=$TCK_HOME/${TCK_NAME}work/${TCK_NAME}#g" ts.jte
 
-mkdir -p $TCK_HOME/cajtckreport/cajtck
-mkdir -p $TCK_HOME/cajtckwork/cajtck
+mkdir -p $TCK_HOME/${TCK_NAME}report/${TCK_NAME}
+mkdir -p $TCK_HOME/${TCK_NAME}work/${TCK_NAME}
 
 cd $TCK_HOME/glassfish5/bin
 ./asadmin start-domain
 
 cd $TS_HOME/src/com/sun/ts/tests/signaturetest/caj
 ant runclient
 
-TCK_NAME=cajtck
 JT_REPORT_DIR=$TCK_HOME/${TCK_NAME}report
```

Comment:
Another nit...  JT_REPORT_DIR could be set earlier, and could be used in more places in the script.
It might also be helpful to define a JT_WORK_DIR similarly.

Reply:
> Another nit... JT_REPORT_DIR could be set earlier, and could be used in more places in the script.
> It might also be helpful to define a JT_WORK_DIR similarly.
Will make that change.

Reply:
> > Another nit... JT_REPORT_DIR could be set earlier, and could be used in more places in the script.
> > It might also be helpful to define a JT_WORK_DIR similarly.


Will make this improvement later.